cpp/src/slice2cpp/CPlusPlusUtil.cpp

```diff
@@ -136,11 +136,11 @@ namespace
             {
                 string_view directive = meta->directive();
                 string_view arguments = meta->arguments();
-                if (directive == "cpp:view-type" && !arguments.empty())
```

Comment:
We should no longer be performing "on-the-spot" metadata validation like this.

We trust that the metadata validator has done it's thing and any invalid metadata was removed.
No need to double check it's work.